src/app/components/home.component.ts

```diff
@@ -768,6 +768,9 @@ export class HomeComponent implements OnInit, AfterViewInit {
       if (settingsObject.remoteSettings) {
         this.remoteSettings = settingsObject.remoteSettings;
       }
+      if (settingsObject.wizardOptions) {
```

Comment:
The interface `SettingsObject` doesn't include `wizardOptions` so this conditional would never evaluate to true.

But I'll merge this PR and just add the wizard options to the settings object when it's written to disk (to persist the user's preference) 👌